templates/views/index.pug

```diff
@@ -9,6 +9,10 @@ block content
 				h1 Forged Alliance Forever
 				h3 The Community Driven Lobby for </br> Supreme Commander: Forged Alliance
 			a(class="btn btn-default btn-lg btn-outro btn-danger", href='/client') Download Client
+			p 
+				| Any issue ? 
+				a(href=client_download_link, style="color:white;text-decoration:underline;") Try the legacy client
```

Comment:
Only link ‚legacy client‘, add ‚.‘ at end of sentence.

Reply:
I don't think it's necessary to add a dot, but i'm going to fix the link

Reply:
Sentences always need punctuation. Don’t you think so?

Reply:
But don’t include the dot in the link.